### DISCUSSION TITLE
Display cover image

### DISCUSSION THREAD

**Discussion by eduard93** [adityatelange/hugo-PaperMod#878] [Q&A] [answered]
I want to add an image as a cover - to be displayed alongside the post on a main page.

I have (simplified):

```markdown
---
title: "test"
date: 2022-02-24
draft: false
cover:
    image: "/0.jpg"
---

start

![image](/0.jpg#center)

end
```

When I open the post I see a rendered `0.jpg` but nothing on the main page.

**Comment by orimdominic** [comment, marked as answer]
This is what I use
```md
cover:
    image: "../path/to/image.png"
    alt: "The alt text"
    caption: ""
    relative: false
```
You may want to check the path again and/or use the relative attribute
See this part of the documentation [Post Cover Image](https://adityatelange.github.io/hugo-PaperMod/posts/papermod/papermod-features/#post-cover-image)

**Reply by socekin** [reply]
![CleanShot 2022-04-19 at 20 02 52@2x](https://user-images.githubusercontent.com/874961/163999065-3d298e53-5669-4f1b-8bf6-729036524a14.png)
I try to add cover as your method, but it can't work. the cover can not display in the home page. but the image can display in the post page

**Reply by adityatelange** [reply]
`image: imgs/cover.png`

**Reply by socekin** [reply]
thanks

**Reply by ahmadpiran** [reply]
@adityatelange 
It also doesn't work for me. It displays the image in the post but not the cover image.
